app/uk/gov/hmrc/cataloguefrontend/leakdetection/LeakDetectionService.scala

```diff
@@ -43,7 +43,7 @@ class LeakDetectionService @Inject() (
     }
 
   private val ldsIntegrationEnabled: Boolean    = configuration.get[Boolean]("lds.integrationEnabled")
-  private val repositoriesToIgnore: Seq[String] = configuration.get[Seq[String]]("lds.noWarningsOn")
+  // private val repositoriesToIgnore: Seq[String] = configuration.get[Seq[String]]("lds.noWarningsOn")
```

Comment:
Is this not used? Can we delete it?

Reply:
It's not being used. I commented it out to fix a warning.